exercises/hamming/canonical-data.json

```diff
@@ -79,6 +79,50 @@
         "strand2": "AGTG"
       },
       "expected": {"error": "left and right strands must be of equal length"}
+    },
+    {
+      "uuid": "5dce058b-28d4-4ca7-aa64-adfe4e17784c",
+      "description": "disallow left empty strand",
+      "property": "distance",
+      "input": {
+        "strand1": "",
+        "strand2": "G"
+      },
+      "expected": {"error": "left strand must not be empty"}
```

Comment:
Why can't it be empty?  As long as the right strand is also empty, representing that it is not there, the distance could still be said to be 0.  And if there is a length of not 0 for the right hand strand, the distance would equal the length of the second strand?  (I will have to go look at the discussions to find out if this is coming from there, though)

Reply:
The case for both strands being empty is allowed as a distance of 0, as you mention, per the case specified on ln 14, but the ambiguity that prompts your question is partly why this change is proposed 😃